core/client/fs/src/main/java/alluxio/client/block/stream/BlockWorkerClientPool.java

```diff
@@ -46,27 +46,33 @@ public final class BlockWorkerClientPool extends DynamicResourcePool<BlockWorker
           ThreadFactoryUtils.build("BlockWorkerClientPoolGcThreads-%d", true));
   private static final Counter COUNTER = MetricsSystem.counter(
       MetricKey.CLIENT_BLOCK_WORKER_CLIENT_COUNT.getName());
-  private final AlluxioConfiguration mConf;
+  private final MasterClientContext mMasterContext;
```

Comment:
I don't think we need to hold on to a reference to a `MasterClientContext` object if all we do is calling `getClusterConf()` on it.